azkaban_cli/azkaban_cli.py

```diff
@@ -380,6 +381,32 @@ def __fetch_flow_execution(ctx, execution_id):
     except FetchFlowExecutionError as e:
         logging.error(str(e))
 
+def __log_flow_execution_updates(json):
+    logging.info('Id: %s' % (json.get('id')))
+    logging.info('Start Time: %s' % (json.get('startTime')))
+    logging.info('Attempt: %s' % (json.get('attempt')))
+    logging.info('Status: %s' % (json.get('status')))
+    logging.info('Update Time: %s' % (json.get('updateTime')))
```

Comment:
Missing `flow` and `endTime` from main response body

Reply:
Add to **azkaban_cli/azkaban_cli.py** 
````
    logging.info('Flow: %s' % (json.get('flow')))
    logging.info('Flow end time: %s' % (json.get('endTime')))
````